src/v2/Apps/Consign/Routes/SubmissionFlow/ArtworkDetails/Components/ArtworkDetailsForm.tsx

```diff
@@ -86,6 +87,7 @@ export interface ArtworkDetailsFormModel {
 }
 
 export const ArtworkDetailsForm: React.FC = () => {
+  const [isAutosuggestError, setIsAutosuggestError] = useState(false)
```

Comment:
`useState` is actually a generic function that can have the type passed into it like this:
```typescript
const [isAutosuggestError, setIsAutosuggestError] = useState<boolean>(false)
```

Reply:
yes, but it is unnecessary here since the type is inferred from its initial state for primitive types.